code/sdl/sdl_input.c

```diff
@@ -1169,18 +1169,19 @@ IN_Frame
 void IN_Frame( void )
 {
 	qboolean loading;
+	qboolean isFullscreen = SDL_GetWindowFlags( SDL_window ) & SDL_WINDOW_FULLSCREEN;
```

Comment:
Why not just fix `cls.glconfig.isFullscreen`?

```c
cls.glconfig.isFullscreen = SDL_GetWindowFlags( SDL_window ) & SDL_WINDOW_FULLSCREEN;
```

Reply:
`cls.glconfig` is only set on renderer initialization (`CL_InitRenderer`), so any change to `glConfig` (in renderer) won't be visible to the client code.

Another possibility would be using `r_fullscreen` but since we're in SDL code, we might as well go the direct way in case the window changes its fullscreen state without the client knowing